configuration/log-config-0.yaml

```diff
@@ -89,7 +89,7 @@ options:
 
 NodeId:
 Protocol: RealPBFT
-GenesisFile: configuration/genesis/genesis.json
+GenesisFile: genesis/genesis.json
```

Comment:
Ah I see why you change the default above. Perhaps it'd be better to move these sets of related example configs into their own subdirs, rather than splitting them like we do now.

e.g.
```
example0/config.yaml
example0/genesis.json
example0/topology.json
```

Reply:
Common file naming within each directory is much preferred @Jimbo4350. Has this been decided?

Reply:
@rhyslbw yes unless somebody is opposed